src/screens/CreateWalletScreen.tsx

```diff
@@ -88,21 +87,41 @@ export class CreateWalletScreen extends React.PureComponent<Props, State> {
     ]);
   };
 
+  getNotificationDescription = () => (
+    <Text style={styles.notificationDescription}>
+      {i18n.notifications.receiveTransactionDescription}
+      <Text style={styles.boldedText}>{this.props.email}</Text>
+    </Text>
+  );
+
   generateWallet = (wallet: Wallet, onError: Function) => {
     const { label } = this.state;
-    const { navigation, createWallet, isNotificationEmailSet, email } = this.props;
+    const { navigation, createWallet, email } = this.props;
     wallet.setLabel(label || i18n.wallets.details.title);
     createWallet(wallet, {
       onSuccess: (w: Wallet) => {
         navigation.navigate(Route.CreateWalletSuccess, {
           secret: w.getSecret(),
-          onButtonPress: isNotificationEmailSet
+          onButtonPress: !!email
             ? () =>
-                this.props.navigation.navigate(Route.ReceiveNotificationsConfirmation, {
-                  address: email, // TODO
-                  flowType: ConfirmAddressFlowType.RECEIVE_NOTIFICATIONS_CONFIRMATION_IMPORT,
+                navigation.navigate(Route.Entity, {
+                  title: i18n.notifications.notifications,
+                  subtitle: i18n.notifications.getNotification,
```

Comment:
![Zrzut ekranu 2021-01-19 o 15 14 57](https://user-images.githubusercontent.com/67273035/105046234-209a9680-5a69-11eb-9c88-2303453c905e.png)
i don't see the email named passed and the bold text